sdk/python/feast/pyspark/launchers/gcloud/dataproc.py

```diff
@@ -265,7 +264,7 @@ def _stage_file(self, file_path: str, job_id: str) -> str:
         return blob_uri_str
 
     def dataproc_submit(
-        self, job_params: SparkJobParameters
+        self, job_params: SparkJobParameters, extra_properties: Dict[str, str]
```

Comment:
I'm not sure if you saw #1198. Should we close that PR after yours has been incorporated, or first merge that one?

Reply:
No i haven't. It looks like it is solving a slightly different problem, namely _user-specified_ extra options. Don't feel strongly which one to merge first (and which one to rebase).